butane: Add version 0.13.0

Okeanos opened the pull request:
This pull request adds an app manifest for [Butane](https://github.com/coreos/butane), a config transpiler for [Fedora CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/) configs.

Okeanos commented:
Fixed the formatting issue, sorry about that.

Ash258 commented on bucket/butane.json:
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.11.0",
+    "description": "Butane translates Butane Configs into Ignition Configs.",
+    "homepage": "https://github.com/coreos/butane/",
```
```suggestion
    "homepage": "https://coreos.github.io/butane/",
```

Ash258 commented on bucket/butane.json:
```diff
@@ -0,0 +1,24 @@
+    },
+    "bin": "butane.exe",
+    "checkver": {
+        "github": "https://github.com/coreos/butane/"
```
```suggestion
        "github": "https://github.com/coreos/butane"
```

Ash258 commented on bucket/butane.json:
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.11.0",
+    "description": "Butane translates Butane Configs into Ignition Configs.",
```
```suggestion
    "description": "Fedora CoreOS Config Transpiler, FCCT. Translates Butane configs into Ignition configs.",
```

Okeanos replied:
The official description is 
>Butane translates human-readable Butane Configs into machine-readable Ignition Configs.

and FCCT should not be used anymore since the rename ([butane#167](https://github.com/coreos/butane/issues/167).

Ash258 replied:
Irrelevant. For searching and reference it should be there

Okeanos replied:
I updated the description to make the history of the project clearer but want to point out that the e.g. [Homebrew formula](https://formulae.brew.sh/formula/butane) received a separately tracked name change and the [MacPorts version](https://ports.macports.org/port/butane/summary) has no reference at all because it was introduced afterwards. I guess what I am trying to say is: the description may not be the right place to put this info?

Ash258 commented on bucket/butane.json:
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.11.0",
+    "description": "Butane translates human-readable Butane configs into machine-readable Ignition configs. Formerly known as Fedora CoreOS Config Transpiler (FCCT).",
```
```suggestion
    "description": "Fedora CoreOS Config Transpiler, FCCT. Translates Butane configs into Ignition configs.",
```

Okeanos replied:
The project isn't called `Fedora CoreOS Config Transpiler (FCCT)` and that shouldn't be the main takeaway from the description. In case of Homebrew, the [formula](https://formulae.brew.sh/formula/butane) received a separately tracked name change but other than that doesn't refer to the outdated name at all and the [MacPorts](https://ports.macports.org/port/butane/summary) version has no reference to it because it was introduced afterwards.

Okeanos replied:
@Ash258 what are we going to do about this? If you absolutely insist on putting the outdated description in … I guess I'll oblige despite my reservations.

Okeanos commented:
To summarize the current state of this pull request: the point of contention is the description of the package.

The project ist called Butane (renamed from Fedora CoreOS Config Transpiler (FCCT)) and everywhere else (that I am aware of) the FCCT name has been replaced or was never introduced in the first place because Butane was introduced later. Only the Butane project pages give an in depth explanation of this.

Examples:
- [MacPorts](https://ports.macports.org/port/butane/summary): introduced after the renaming and has no reference to FCCT
- [Homebrew](https://formulae.brew.sh/formula/butane): introduced before the renaming and was officially renamed with a redirect attached.
- [Official Fedora CoreOS Documentation](https://docs.fedoraproject.org/en-US/fedora-coreos/producing-ign/): makes no mention of FCCT but links to Butane project pages and uses Butane wording.

Based on that I would argue for the following description:

> Butane translates human-readable Butane configs into machine-readable Ignition configs. Formerly known as Fedora CoreOS Config Transpiler (FCCT).

If that is not acceptable (based on current feedback) I'll accept

> Fedora CoreOS Config Transpiler, FCCT. Translates Butane configs into Ignition configs.

with a sad heart just to get it over with and this pull request merged.

Okeanos commented:
@Ash258 I really don't want to annoy you too much but could you give a definitive answer on what I have to fix so this'll get merged, please?

Okeanos commented:
@Ash258 I applied your code suggestion. Could you please tell me whether there's anything else that needs to be done here?

Ash258 commented:
/verify
